packages/pixeloven-webpack-compiler/README.md

```diff
@@ -0,0 +1,52 @@
+# @pixeloven/webpack-compiler
+
+> Pixel Oven webpack-compiler.
+
+See our website [pixeloven-webpack-compiler](https://github.com/pixeloven/pixeloven) for more information or the [issues](https://github.com/pixeloven/pixeloven) associated with this package.
+
+## Install
+
+Using npm:
+
+```sh
+npm install --save @pixeloven/webpack-compiler
+```
+
+or using yarn:
+
+```sh
+yarn add @pixeloven/webpack-compiler
+```
+
+## Usage
+> Note It is required that we provide a "client" and "server" configuration provide as an array.
```

Comment:
Nit: second "provide" is redundant and could maybe use clarity by adding "as a single array"

Reply:
updated